XML/Other/NetworkSecurityGroupRules.xml

```diff
@@ -0,0 +1,49 @@
+<!--# We define inbound and outbound security rules for Windows VM or Linux VM in this file.-->
+<platform>
+    <linux>
+        <rule>
+            <name>CirrusLisav2SecurityRule</name>
+            <properties>
+                <description>CirrusLisav2SecurityRule</description>
+                <protocol>tcp</protocol>
+                <sourcePortRange>*</sourcePortRange>
+                <destinationPortRange>22</destinationPortRange>
+                <sourceAddressPrefix>10.0.0.4/32</sourceAddressPrefix>
```

Comment:
Can you post the detail information of 'sourceAddressPrefix'; what it is for, how it is working?

Reply:
This mask is just give an IP scope that will be allowed or denied following this rule. /32 means only the current IP itself, in real practice, the IP and mask will be replaced in the xml here as needed.